src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -170,7 +173,7 @@ public void stopFeeder() {
 
     // Get and Set shooter states
     public void startShooter(double rpms) {
-        shooter.set(TalonFXControlMode.Velocity, rpms * 2048 / 10);
+        shooter.set(TalonFXControlMode.Velocity, rpms * 2048 / 10 * 60);
```

Comment:
Divide by 60?

Reply:
?

Reply:
I think you need to divide by 60 instead of multiplying. 

(Rev/min) * (ticks/rev) is (ticks/min). If there were 60 ticks per minute, that would be 1 per second, not 3600.